test/test_resources.py

```diff
@@ -72,5 +72,10 @@ def test_get(self):
     def test_open(self):
         rm = ResourceManager()
         rm.add_path(__file__)
-        fp = rm.open(__file__)
-        self.assertEqual(fp.read(), open(__file__).read())
+        given_fp = rm.open(__file__)
+        expected_fp = open(__file__)
+        try:
+            self.assertEqual(given_fp.read(), expected_fp.read())
+        finally:
+            given_fp.close()
+            expected_fp.close()
```

Comment:
Looks good but does not belong in commit "Update deprecated from 2.7 assertEquals", in my opinion.